### PR TITLE
Nanmin and nanmax

### PR DESCRIPTION
**Pull request by unnonouno**
I implemented nanmin and nanmax.
related to #1536

### CODE REVIEW

**Comment by unnonouno** [comment]
ping

**Comment by unnonouno** [comment]
fixed

**Comment by ir5** [comment]
Looks good to me.

File: cupy/statistics/order.py [hunk 1]
```diff
@@ -52,10 +58,56 @@ def amax(a, axis=None, out=None, keepdims=False, dtype=None):
     return a.max(axis=axis, dtype=dtype, out=out, keepdims=keepdims)
 
 
-# TODO(okuta): Implement nanmin
+def nanmin(a, axis=None, out=None, keepdims=False):
+    """Return the minimum of an array along an axis ignoring NaN.
```
Comment (beam2d) [inline cupy/statistics/order.py]: Return"s"

File: cupy/statistics/order.py [hunk 2]
```diff
@@ -52,10 +58,56 @@ def amax(a, axis=None, out=None, keepdims=False, dtype=None):
...
 
+    When all NaN slices are found, a :class:`RuntimeWarning` is raised and NaN
```
Comment (beam2d) [inline cupy/statistics/order.py]: Does "all Nan slices" mean a slice whose elements are all NaN? If so, rewrite it as `When there is a slice whose elements are all NaN, ...`.

File: cupy/statistics/order.py [hunk 3]
```diff
@@ -52,10 +58,56 @@ def amax(a, axis=None, out=None, keepdims=False, dtype=None):
...
+    """
+    res = core.nanmin(a, axis=axis, out=out, keepdims=keepdims)
+    if content.isnan(res).any():
+        warnings.warn('All-NaN slice encountered', RuntimeWarning)
+    return res
 
-# TODO(okuta): Implement nanmax
+
+def nanmax(a, axis=None, out=None, keepdims=False):
+    """Return the maximum of an array along an axis ignoring NaN.
```
Comment (beam2d) [inline cupy/statistics/order.py]: Return"s"

File: cupy/statistics/order.py [hunk 4]
```diff
@@ -52,10 +58,56 @@ def amax(a, axis=None, out=None, keepdims=False, dtype=None):
...
+
+    When all NaN slices are found, a :class:`RuntimeWarning` is raised and NaN
```
Comment (beam2d) [inline cupy/statistics/order.py]: see the comment above

**Comment by beam2d** [comment]
Thanks!

**Pull request merged**